scripts/check_duplicates.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+# Copyright (c) 2017, Open Source Robotics Foundation
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     * Redistributions of source code must retain the above copyright
+#       notice, this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in the
+#       documentation and/or other materials provided with the distribution.
+#     * Neither the name of the Willow Garage, Inc. nor the names of its
+#       contributors may be used to endorse or promote products derived from
+#       this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+import yaml
+
+# pretty - A miniature library that provides a Python print and stdout
+# wrapper that makes colored terminal text easier to use (eg. without
+# having to mess around with ANSI escape sequences). This code is public
+# domain - there is no license except that you must leave this header.
+#
+# Copyright (C) 2008 Brian Nez <thedude at bri1 dot com>
+#
+# With modifications
+#           (C) 2013 Paul M <pmathieu@willowgarage.com>
+
+codeCodes = {
+    'black':     '0;30',     'bright gray':   '0;37',
+    'blue':      '0;34',     'white':         '1;37',
+    'green':     '0;32',     'bright blue':   '1;34',
+    'cyan':      '0;36',     'bright green':  '1;32',
+    'red':       '0;31',     'bright cyan':   '1;36',
+    'purple':    '0;35',     'bright red':    '1;31',
+    'yellow':    '0;33',     'bright purple': '1;35',
+    'dark gray': '1;30',     'bright yellow': '1;33',
+    'normal':    '0'
+}
+
+
+def printc(text, color):
+    """Print in color."""
+    if sys.stdout.isatty():
+        print("\033["+codeCodes[color]+"m"+text+"\033[0m")
+    else:
+        print(text)
+
+
+def print_test(msg):
+    printc(msg, 'yellow')
+
+
+def print_err(msg):
+    printc('  ERR: ' + msg, 'red')
```

Comment:
I would rather avoid adding this colorization logic just for this test. None of the other tests does it either.